resolwe_api/resolwe.py

```diff
@@ -211,18 +237,24 @@ def print_process_inputs(self, process_name):
 
         if len(p) == 1:
             p = p[0]
+        elif len(p) > 1:
+            # Multiple processors with same slug, but different versions:
+            versions = ([int(x['version']) for x in p])
+            # Get index of the latest processor version:
+            top_index = sorted(enumerate(versions), key=lambda x:x[1])[-1][0]
+            p = p[top_index]
         else:
-            Exception('Invalid process name')
+            raise Exception('Invalid process name: {}'.format(process_name))
 
         for field_schema, _, _ in iterate_schema({}, p['input_schema'], 'input'):
             name = field_schema['name']
             typ = field_schema['type']
-            # value = fields[name] if name in fields else None
-            print("{} -> {}".format(name, typ))
+            sys.stdout.write("{} -> {}".format(name, typ))
 
     def rundata(self, strjson):
```

Comment:
Remove this function. We will probably want a similar function in the future, _e.g._ `run(process_name, inputs)` that will return a data object. This would be similar to the `create` function, but more explicit that it runs the process in the backend. Anyway, let's remove it for now and wait for the bioinformatician's workflows.